### PR TITLE
N14: femtovg

### PR DESCRIPTION
**Pull request by adamnemecek**
_Part of #278 ("Newsletter 14")_

### CODE REVIEW

File: content/posts/newsletter-014/index.md [hunk 1]
```diff
@@ -66,6 +66,24 @@ If needed, a section can be split into subsections with a "------" delimiter.
...
+for contributors.
+
+Unlike NanoVG, FemtoVG has full text-shaping support thanks to harfbuzz.
+
+FemtoVG, just like the original NanoVG, is based on the _stencil-then-cover_
+approach presented in [GPU-accelerated Path Rendering](https://github.com/femtovg/femtovg/blob/master/assets/gpupathrender.pdf).
+
+Join the [Discord channel](https://discord.gg/V69VdVu) or follow [FemtoVG on twitter](https://twitter.com/femtovg).
+
+
```
Comment (17cupsofcoffee) [inline content/posts/newsletter-014/index.md:86]:
```suggestion
```

**Review by 17cupsofcoffee — APPROVED** [review]
Content looks good, just needs an extra newline removing to fix the linter errors.

**Review by ozkriff — APPROVED** [review]
:+1: (made a few fmt tweaks for consistency)

**Comment by ozkriff** [comment]
Merged, thanks for the PR!

**Pull request merged**